test/functional/apps/dashboard/_dashboard.js

```diff
@@ -14,127 +14,93 @@ import {
 import PageObjects from '../../../support/page_objects';
 
 bdd.describe('dashboard tab', function describeIndexTests() {
-  bdd.before(function () {
-    PageObjects.common.debug('Starting dashboard before method');
-    const logstash = scenarioManager.loadIfEmpty('logstashFunctional');
-    // delete .kibana index and update configDoc
-    return esClient.deleteAndUpdateConfigDoc({ 'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*' })
-    // and load a set of makelogs data
-      .then(function loadkibanaVisualizations() {
-        PageObjects.common.debug('load kibana index with visualizations');
-        return elasticDump.elasticLoad('dashboard','.kibana');
-      })
-      .then(function () {
-        PageObjects.common.debug('navigateToApp dashboard');
-        return PageObjects.common.navigateToApp('dashboard');
-      })
-      // wait for the logstash data load to finish if it hasn't already
-      .then(function () {
-        return logstash;
-      });
+  bdd.before(async function () {
+    return PageObjects.dashboard.initTests();
   });
 
-  bdd.describe('add visualizations to dashboard', function dashboardTest() {
-    const visualizations = ['Visualization漢字 AreaChart',
-      'Visualization☺漢字 DataTable',
-      'Visualization漢字 LineChart',
-      'Visualization PieChart',
-      'Visualization TileMap',
-      'Visualization☺ VerticalBarChart',
-      'Visualization MetricChart'
-    ];
+  bdd.it('should be able to add visualizations to dashboard', async function addVisualizations() {
+    PageObjects.common.saveScreenshot('Dashboard-no-visualizations');
 
-    bdd.it('should be able to add visualizations to dashboard', function addVisualizations() {
-      PageObjects.common.saveScreenshot('Dashboard-no-visualizations');
-
-      function addVisualizations(arr) {
-        return arr.reduce(function (promise, vizName) {
-          return promise
-            .then(function () {
-              return PageObjects.dashboard.addVisualization(vizName);
-            });
-        }, Promise.resolve());
-      }
-
-      return PageObjects.dashboard.clickNewDashboard()
-        .then(() => addVisualizations(visualizations))
-        .then(function () {
-          PageObjects.common.debug('done adding visualizations');
-          PageObjects.common.saveScreenshot('Dashboard-add-visualizations');
-        });
-    });
+    // This flip between apps fixes the url so state is preserved when switching apps in test mode.
```

Comment:
@LeeDr I just want to bring your attention to this comment.  I'm hoping the ?_t=#### isn't actually necessary for testing?  Having it in the url was breaking my dark theme test below.